packages/next-s3-upload/src/hooks/use-upload-files.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import { FileInput } from '../components/file-input';
 import { useRef, useState } from 'react';
+import { uuid } from 'next-s3-upload/src/utils/keys';
```

Comment:
Let's do relative import since I believe the rest of the code is doing this. Absolute might be fine but let's do relative to keep things consistent.

Reply:
Got it